src/components/configure/customkey/AutocompleteKeys.tsx

```diff
@@ -5,6 +5,29 @@ import { TextField } from '@material-ui/core';
 import { IKeymap } from '../../../services/hid/Hid';
 import { KeymapCategory } from '../../../services/hid/KeycodeList';
 
+/**
+ * Filter and sort strategy.
+ * CASE INSENSITIVE
+ * 1st priority: Match a key's label.
+ * 2nd priority: Match a kinds.
+ */
+const filterOptions = (
+  options: IKeymap[],
+  { inputValue }: { inputValue: string }
+) => {
+  const value = inputValue.toLowerCase();
+  const matcheLabels = options.filter(
```

Comment:
@adamrocker Probably, the variable name should be `matchedLabels`.

Reply:
@yoichiro Sure!